libcodechecker/libhandlers/analyze.py

```diff
@@ -176,8 +176,8 @@ def is_ctu_capable():
 
     analyzer_opts.add_argument('--add-compiler-defaults',
                                action='store_true',
-                               default=False,
                                required=False,
+                               default=argparse.SUPPRESS,
```

Comment:
Why do we need this change?

Reply:
To not have `(default: False)` printed for the help message in every argument if the argument is already a boolean, and its help already makes it apparent that "it does _something_ if given, otherwise it does not do _something_ ".